docs/Configuration.rst

```diff
@@ -2828,7 +2831,31 @@ To disable the activity type, simply set the parameter to ``false``.
 			   a content is shared in a space.  
 			-  **exosocial\\:relationship**: Activity post when two user are
 			   connected together.
+		
+.. _Configuration.ActivityLinkPreview:		
+		
 			    
+==========================================
+Enabling/Disabling activity link preview
+==========================================
+
+eXo Platform allows you to enable or disable the link preview in an activity 
```

Comment:
We should add an little introduction to describe what is link preview IMO (maybe there is already a documentation page for that ?), something like : "when posting an activity with a link attached, the activity display a preview for this link by fetching the title, an excerpt and a thumbnail of the linked resource", followed by a screenshot showing an example (with the same link than in the other screenshot to show the difference). WDYT ?

Reply:
Yep I like the idea, I will update this part

Reply:
Updated